packages/expo-notifications/android/src/main/java/expo/modules/notifications/Utils.kt

```diff
@@ -17,3 +26,77 @@ internal fun createDefaultResultReceiver(
     }
   }
 }
+
+/**
+ * Returns true if object is convertible to a WritableMap element
+ */
+internal fun isWritableMapConvertible(value: Any?): Boolean {
+  return when (value) {
+    null, is Unit -> true
+    is Bundle -> true
```

Comment:
what happens if this bundle has a nested element that can't be serialized? (I think anything here needs to be recursive)

Reply:
The recursiveness is handled in `filteredBundleForWritableMap` below.